app/assets/stylesheets/application.scss

```diff
@@ -4,6 +4,7 @@ $govuk-use-legacy-palette: false;
 @import 'govuk_publishing_components/govuk_frontend_support';
 @import 'govuk_publishing_components/component_support';
 @import 'govuk_publishing_components/components/breadcrumbs';
+@import 'govuk_publishing_components/components/button'; // not in application but needed for cookie banner
```

Comment:
Ideally, the cookie banner component will contain `@import 'govuk_publishing_components/components/button';` but Ruby packages for scss/sass compilation seem to transform any sort of nesting in file imports into a painful compilation.

Reply:
Indeed. We have a lot of components that contain other components, I'm looking forward to coming up with a better solution to all of this in the future.